webpack.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = (env, argv) => {
             library: dashLibraryName,
             libraryTarget: 'window',
         },
+        devtool: 'source-map',
```

Comment:
Source map! This setting is slower but more accurate than other mappings. See here: https://webpack.js.org/configuration/devtool/

Reply:
Need to be sure I don't lose my debugging support with this, I remember the plain source-maps did not work so well.

Reply:
Is this the in-IDE support you mentioned to me a little while ago?

Reply:
Yes, I'll try it to be sure.

Reply:
Thanks. If it doesn't work I'll change it to:
mode === 'development' ? 'eval-source-map' : 'source-map'
which will give you the old behavior in dev